config/nginx.conf.erb

```diff
@@ -284,6 +284,10 @@ http {
                   return 302 https://peps.python.org/;
                 }
 
+                location ~ ^/sf/(.*)$ {
+                  return 302 https://bugs.python.org/issue$1;
+                }
+
                 location ~ ^/sf(.*)$ {
```

Comment:
nginx uses the first match, so this location block is now redundant and should be removed.